x-pack/legacy/plugins/maps/public/connected_components/layer_panel/_layer_panel.scss

```diff
@@ -14,7 +14,7 @@
   padding-bottom: 0; /* 1 */
   flex-grow: 1;
   flex-basis: 1px; /* Fixes scrolling for Firefox */
-  overflow-y: hidden;
+  overflow-y: auto;
 
   .mapLayerPanel__bodyOverflow {
     @include euiScrollBar;
```

Comment:
So it looks like you won't be able to use the `euiOverflowShadow` mixin which is totally fine because you at least have borders indicating the top and bottom bounds of the flyout body. Therefore you've some things you can remove now.

From this selector remove the first three properties scrollbar mixin, `height`, and `overflow`.